src/azure-cli-core/azure/cli/core/auth/binary_cache.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import collections.abc as collections
```

Comment:
Renaming built-in names is a bad idea. I don't really remember why I wrote this line in https://github.com/Azure/azure-cli/pull/20847. 😂